airbyte-webapp/src/core/env.ts

```diff
@@ -0,0 +1 @@
+export const IS_CLOUD = process.env.REACT_APP_CLOUD || window.CLOUD === "true";
```

Comment:
We don’t need any env.ts files as we already have config mechanism for specifying the config